redhat-cop/monitoring
# Bind exporter

### What does this PR do?
Add role to deploy bind_exporter on target hosts. The role open's `"{{ bind_exporter_port }}"` in firewall. The stats page has to be available on target host. Prometheus.yml in setup-prometheus role was updated accordingly in PR #9. 

### How should this be tested?
Run the role and then check target host. The bind_exporter pod should be running.
Command below should return valid metrics.
```
curl <target_host>:{{ bind_exporter_port }}/metrics |grep bind
```

### Is there a relevant Issue open for this?
https://gitlab.consulting.redhat.com/rht-labs/labs-sre/documentation/-/issues/404

### Other Relevant info, PRs, etc.
https://github.com/redhat-cop/monitoring/pull/9

### People to notify
cc: @redhat-cop/monitoring
